custom_code/mmt.py

```diff
@@ -191,32 +193,25 @@ class MMTMMIRSSpectroscopyForm(MMTBaseObservationForm):
     filter = forms.ChoiceField(choices=[('zJ', 'zJ'), ('HK', 'HK')])
     grism = forms.ChoiceField(choices=[('J', 'J'), ('HK', 'HK'), ('HK3', 'HK3')])
     gain = forms.ChoiceField(choices=[
-        ('Low noise (0.95)', 'Low noise (0.95)'),
-        ('High Dynamic Range (2.68)', 'High Dynamic Range (2.68)')
+        ('low', 'low'),
+        ('high', 'high')
     ])
     read_tab = forms.ChoiceField(choices=[
         ('ramp_1.475', 'ramp_1.475'),
         ('ramp_4.426', 'ramp_4.426'),
         ('fowler', 'fowler')
     ])
     dither_size = forms.FloatField()
-    slit_width = forms.ChoiceField(choices=[
-        ('0.2 (1pix)', '1pixel'),
-        ('0.4 (2pix)', '2pixel'),
-        ('0.6 (3pix)', '3pixel'),
-        ('0.8 (4pix)', '4pixel'),
-        ('1.0 (5pix)', '5pixel'),
-        ('1.2 (6pix)', '6pixel'),
-        ('2.4 (12pix)', '12pixel'),
-    ])
+    slit_width = forms.ChoiceField(choices=[('1pixel','1pixel'),('2pixel','2pixel'),('3pixel','3pixel'),('4pixel','4pixel'),('5pixel','5pixel'),('6pixel','6pixel'),('12pixel','12pixel')])
```

Comment:
It would be better if we can give the width in arcseconds as the "display name". Do we know what those are? The tuples here are `[(actual_value1, display_name1), (actual_value2, display_name2), ...]`.